src/OpenLoco/src/Windows/Options.cpp

```diff
@@ -1508,6 +1509,9 @@ namespace OpenLoco::Ui::Windows::Options
             // Reloading the objects will force objects to load the new language
             ObjectManager::reloadAll();
             Gfx::invalidateScreen();
+
+            // Rebuild the scenario index to use the new language.
+            ScenarioManager::loadIndex(1);
```

Comment:
Could you name this parameter should really be something like `ForceRebuild::True` and then default it to `ForceRebuild::False`